templates/error.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Build error</title>
+    <script src="/ember-cli-live-reload.js" type="text/javascript"></script>
```

Comment:
Adding some customization hook for ember-cli to load scripts here sounds great. I'd like to land the `.rebuild` API, and then redo the builder, watcher, and server on top. Is it OK to wait till then?